functions/proxy/proxy.js

```diff
@@ -2,19 +2,20 @@
 const fetch = require("node-fetch");
 const { URL, URLSearchParams } = require("url");
 
-exports.handler = async function(event, context) {
+exports.handler = async function (event, context) {
     const { GENIUS_AUTH_TOKEN } = process.env;
 
     const urlString = event.path.replace("/.netlify/functions/proxy/", "");
-    const url = new URL(`https://api.genius.com/${urlString}`);
+    // const url = new URL(`https://api.genius.com/${urlString}`);
+    const url = new URL(`https://genius.com/api/${urlString}`);
     url.search = new URLSearchParams(event.queryStringParameters);
 
     try {
         const response = await fetch(url, {
             headers: {
                 Accept: "application/json",
-                Authorization: `Bearer ${GENIUS_AUTH_TOKEN}`
-            }
+                // Authorization: `Bearer ${GENIUS_AUTH_TOKEN}`,
```

Comment:
Do we not need this token any more?

Reply:
Oh yeah, this was me messing with the different API. Which doesn't appear to need a token but also isn't really public, not sure what the best option here is but the new one lets you search by artist and song separately.

Reply:
Ah okay, well I guess innocent until proven guilty! Let's mess with the new one until we get stopped by the fuzz.